service/db/src/main/java/gov/va/vro/service/db/SaveToDbServiceImpl.java

```diff
@@ -30,6 +40,57 @@ public Claim insertClaim(Claim claim) {
     return claim;
   }
 
+  @Override
+  public AssessmentResult insertAssessmentResult(
+      UUID id, String assessmentResult, String veteranIcn, String diagnosticCode)
+      throws NoSuchFieldException {
+    ClaimEntity claimEntity =
+        claimRepository
+            .findById(id)
+            .orElseThrow(
+                () ->
+                    new NoSuchFieldException(
+                        "Could not match claim ID {" + id.toString() + "} in DB."));
+    Map<String, Object> evidence = new HashMap<>();
+    try {
+      evidence = objMapper.readValue(assessmentResult, Map.class);
+      if (evidence.isEmpty()) {
+        throw new NoSuchFieldException("Could not map assesmentResult, evidence summary is null.");
+      }
+    } catch (Exception e) {
+      log.error("Could not map assesmentResult, evidence summary is null.");
+    }
+    Map<String, Object> defaultEvidence = new HashMap<>();
+    defaultEvidence.put("medicationsCount", 0);
+    Map summary = (Map) evidence.getOrDefault("evidenceSummary", defaultEvidence);
+    Object code = evidence.get("diagnosticCode");
+    String dcode = (String) code;
+    AssessmentResultEntity assessmentResultEntity = new AssessmentResultEntity();
+    assessmentResultEntity.setEvidenceCountSummary(summary);
+    ContentionEntity contention = findContention(claimEntity, diagnosticCode);
+    contention.addAssessmentResult(assessmentResultEntity);
+    claimRepository.save(claimEntity);
+    AssessmentResult resp = new AssessmentResult();
+    resp.setEvidenceSummary(assessmentResultEntity.getEvidenceCountSummary());
+    resp.setVeteranIcn(veteranIcn);
+    resp.setDiagnosticCode(diagnosticCode);
+    return resp;
```

Comment:
This return value is not being used. Which is the right thing to do I believe. But then why are we returning this? Also we can probably have this object in the signature of this function instead of string..